src/session/get-access-token.ts

```diff
@@ -226,13 +229,16 @@ export type GetAccessToken = (
  * @ignore
  */
 export default function accessTokenFactory(
-  config: NextConfig,
-  client: AbstractClient,
+  getConfig: GetConfig,
+  getClient: GetClient,
   sessionCache: SessionCache
 ): GetAccessToken {
   return async (reqOrOpts?, res?, accessTokenRequest?): Promise<GetAccessTokenResult> => {
     const options = (res ? accessTokenRequest : reqOrOpts) as AccessTokenRequest | undefined;
     const req = (res ? reqOrOpts : undefined) as IncomingMessage | NextApiRequest | undefined;
+    // TODO: clean up
```

Comment:
Should this be removed or is there work to do?

Reply:
Ah yep, will remove that comment

Reply:
ed41013